### ISSUE TITLE
Any way i can use it on one screen ?

### ISSUE THREAD

**Issue by xXRainbowXx24** [snoww/loa-logs#116]
Exits any way i can run meter on the same screen as the game becuase i play the game in boderless and activate clickthrouhge but when i click in my gamewindow the meter disappears

**Comment by snoww** [comment]
are you sure the game is in borderless mode? theres an always on top setting in the meter settings, make sure thats also turned on.

**Comment by xXRainbowXx24** [comment]
> are you sure the game is in borderless mode? theres an always on top setting in the meter settings, make sure thats also turned on.

yes i have that on like i said it above but still when i click in game the window gose in the backround

**Comment by snoww** [comment]
you only mentioned clickthrough which is a different setting. but if you do have the always on top mode enabled and its not working, i don't know why.
